mlir/test/Dialect/LLVMIR/canonicalize.mlir

```diff
@@ -101,6 +101,18 @@ llvm.func @fold_bitcast2(%x : i32) -> i32 {
 
 // -----
 
+// CHECK-LABEL: fold_bitcast_chain
+// CHECK-SAME: %[[a0:arg[0-9]+]]
```

Comment:
nit: I would suggest to switch to [[ARG0:[[:alnum:]]+]] rather than matching arg explicitly.

Reply:
+1, there is no guarantee that the name starts with `arg` similar to any other SSA name.

Reply:
Yeah, I copied this from the other tests in the same file. Now all occurrences of this have been adopted in the file.